src/etc/inc/interfaces.inc

```diff
@@ -3772,21 +3772,20 @@ function guess_interface_from_ip($ipaddress)
         /* create a route table we can search */
         exec("/usr/bin/netstat -rnWf inet", $output, $ret);
         foreach ($output as $line) {
-            if (preg_match("/^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+\/[0-9]+[ ]+link[#]/", $line)) {
```

Comment:
interfaces.inc changes should probably not be part of this PR as it is used outside of dhcp leases page as well. oddly enough it is not used in the v4 leases page.

Reply:
I have nothing against pulling these changes separately. However, some DHCPv6 leases (e.g. /120 ones) will not be shown correctly unless the interfaces.inc file is fixed.

Reply:
I'm saying its easier to pull in fixes first before working on GUI representation...